app/views/RoomInfoView/styles.js

```diff
@@ -45,8 +55,41 @@ export default StyleSheet.create({
 		bottom: -4,
 		right: -4
 	},
+	followLabel: {
+		padding: 10,
+		fontSize: 12,
+		...sharedStyles.textColorNormal,
+		...sharedStyles.textMedium
+	},
+	followingContainer: {
+		borderColor: '#f5f5f5',
```

Comment:
Add '#f5f5f5' to Rocket.Chat.ReactNative/app/constants/colors.js as a defined color constant